db/test/versions/0056_test.js

```diff
@@ -0,0 +1,6 @@
+const testing = require('taskcluster-lib-testing');
+
+suite(testing.suiteName(), function() {
+// this version only adds a method,
+// so there is nothing to test on upgrade
```

Comment:
It does add two columns :)

But, I think we will end up squashing all these DB revisions before merging to main anyway, so this is fine.